addon/styles/_frost-link.scss

```diff
@@ -214,10 +215,69 @@ $frost-color-link-focus-box-shadow: 0 1px 6px rgba(0,158,239, 0.75);
     }
 
     .svg {
-      margin-left: 10px;
+      &:not(.action) {
+        margin-left: 10px;
+      }
       .frost-icon {
         vertical-align: middle;
       }
     }
   }
+
+  // Action button styling
+  &.action {
```

Comment:
If this is 'design' style, then I think we start using aliases - so if multiple things use the same design they still get the use case name (e.g. info-bar-action, some-other-case can both point to the same design) - I think this one should be info-bar-action